desktop/cmp/input/CodeInput.scss

```diff
@@ -46,40 +51,41 @@
     position: relative;
     flex: 1;
 
-    & > .CodeMirror {
-      position: absolute;
+    & > .cm-editor {
+      position: absolute !important;
```

Comment:
The latest CSS for code mirror added !important to some of their internal styling. For .cm-editor it made the position relative !important, breaking our needed override here.